packages/synapse-react-client/src/lib/style/components/_access-requirement-list.scss

```diff
@@ -69,6 +69,7 @@
     background-color: SRC.$primary-action-color;
     padding: 10px 70px 10px 70px;
     color: white;
+    border-radius: SRC.$button-border-radius;
```

Comment:
Adjust border-radius for buttons in Access Requirements List, missed in [PORTALS-2597](https://sagebionetworks.jira.com/browse/PORTALS-2597)

Before: 
![main_AccessList](https://user-images.githubusercontent.com/26949006/232629535-b3633897-1391-480d-8f49-e1b6c454491f.png)

After: 
![feature_AccessList](https://user-images.githubusercontent.com/26949006/232629521-a66d9941-4dc7-4bd5-8d32-0ada457187b8.png)